### ISSUE TITLE
Buttons with outlines do not look correct in the editor (but fine on front end)

### ISSUE THREAD

**Issue by oldrup** [mrwweb/mrw-web-design-simple-tinymce#91]
Hey!

I've noticed this odd behavior in the block editor, with WP 6.6 and the Blocksy Theme.

By default my buttons, with or with outline, looks fine:
![image](https://github.com/user-attachments/assets/e4f1ac66-3b11-4daf-a92a-1059b2379a2a)

When I enable Simplified Editor, the button with the outline no longer looks right:
![image](https://github.com/user-attachments/assets/dfcdd401-62eb-46de-b696-c73745c47fb6)

We cannot choose the button styles thus outline with MRW enabled, but some may have been set to outline in a pattern or previously edited page.

Is that something you experience too?

All the best.
Bjarne

**Comment by mrwweb** [comment]
Interesting. I haven't seen this before, and without looking at an example yet, I'm initially surprised this is happening. I'm sure there's a reason!

Could you provide specific steps to reproduce including the block markup in the screenshot?

**Comment by oldrup** [comment]
Sure! Looks like it's not a common issue then -- I'll do some troubleshooting on a demo site that isn't affected by other plugins etc.

**Comment by oldrup** [comment]
Did some further testing on a fresh install.

https://test.oldrup.dev/mrw/sample-page/

It looks like it is theme dependent, in the way, that it works fine with themes Greyd and Kadence (screenshots)
![image](https://github.com/user-attachments/assets/37e59085-2c50-4fa1-82d5-e9f13c8cba2f)

![image](https://github.com/user-attachments/assets/64f18abb-c3d3-4e04-81d3-03086baa78bd)

But fails in Blocksy and Ollie

![image](https://github.com/user-attachments/assets/a0a97a76-0748-4dd3-b094-0410215e34d5)

![image](https://github.com/user-attachments/assets/7dadfa4b-bab8-4353-9d37-1540fa012f47)

Here's the entire button block copied:
```
<!-- wp:buttons -->
<div class="wp-block-buttons"><!-- wp:button -->
<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">This is a normal button</a></div>
<!-- /wp:button -->

<!-- wp:button {"className":"is-style-outline"} -->
<div class="wp-block-button is-style-outline"><a class="wp-block-button__link wp-element-button">This is an outlined button</a></div>
<!-- /wp:button --></div>
<!-- /wp:buttons -->
```

Would a temporary login be helpful? If so, I can share the site with you.

I can also dig a bit into the CSS causing this behavior, but I'm a bit short of time just now.

Happy coding!
Bjarne

**Comment by mrwweb** [comment]
Thanks, @oldrup ! That should be enough detail to reproduce. I'll let you know if I run into other questions. Really appreciate it.

**Comment by mrwweb** [comment]
@oldrup This issue is old but not forgotten! I have reviewed the problem, and I'm not sure this is solveable from the plugin side.

Here's what I think is happening:

1. MRW Simplified Editor hides all default block styles in WP, that includes "Outline" (you can even see this in the screenshot you shared where "Outline" isn't a listed style!)
2. When the block style is unregistered, it impacts how block styles are loaded in the editor

I suspect that style loading changes in WP 6.7 and that's probably why this popped up then. It probably works in some themes but not others depending on how the styles for the outline style are added (inline style from `register_block_style` vs. `.is-style-outline` compiled into the theme's primary stylesheet).

The foundational philosophy of this plugin is that you should show the features you use and hide the features you don't. Restoring the outline block style also restores the style:

```php
add_filter( 'mrw_hidden_block_styles', 'mrw_simplified_editor_restore_button_outline_style' );
function mrw_simplified_editor_restore_button_outline_style( $styles ) {

	$styles['core/button'] = array_diff(
		$styles['core/button'],
		array( 'outline')
	);

	return $styles;

}
```

I'm going to close this issue for now, but would love to hear any feedback you have.

**Comment by oldrup** [comment]
I agree to that decision. If the button outline is important to me, I can restore it using the filter just as I tweak other settings to my needs. 😊

**Comment by mrwweb** [comment]
Glad that makes sense. Appreciate the confirmation.
